### PR TITLE
Just so you know [#172593056]

### PR DESCRIPTION
**Pull request by MatthewSchultz**
Add a "Just so you know" page following children entry. Page should have basic text, and no input. Copy/details on Figma.

### CODE REVIEW

File: app/views/just_so_you_know/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      <div class="emoji emoji--bigger spacing-below-25 spacing-below-25 emoji--credit-card"></div>
+      <h2 class="form-card__title">
+        Just so you know.
+      </h2>
+    </div>
+    <div class="form-card__content">
+      <div class="grid">
+        <div class="grid__item">
+          <p class="with-padding-small text--left">
+            The card will have the oldest student&apos;s name on it when it arrives. Adults in the household will be able to use it to shop for the whole family.
```
Comment (admande) [inline app/views/just_so_you_know/edit.html.erb:15]: This is missing the second sentence of the copy from Figma

File: spec/controllers/just_so_you_know_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe EligibleController do
```
Comment (admande) [inline spec/controllers/just_so_you_know_controller_spec.rb:3]: This should be `JustSoYouKnowController`

**Comment by admande** [comment]
Will you be squashing/rebasing commits prior to merge?

File: app/views/just_so_you_know/edit.html.erb (hunk 1 above)
Comment (MatthewSchultz) [reply app/views/just_so_you_know/edit.html.erb]: Looks like the copy for this was changed over the weekend ... and not just the additional sentence. New copy added. Also, the copy from Figma is slightly incorrect: it references the `oldest students' name` where it should be the `oldest student's` name. Student, in this case, is specifically singular (there can be only one oldest student), and therefore, the singular form of the possessive is desired here.

Changes added to PR.

File: spec/controllers/just_so_you_know_controller_spec.rb (hunk 2 above)
Comment (MatthewSchultz) [reply spec/controllers/just_so_you_know_controller_spec.rb]: Yes, yes it should. Changes added to PR.

**Comment by MatthewSchultz** [comment]
> Will you be squashing/rebasing commits prior to merge?

Yeah, it'll need to pull in master / do tests again before final merge (as is the case with every PR).

**Pull request merged**